src/Artsy/Analytics/trackingMiddleware.ts

```diff
@@ -72,7 +72,7 @@ export function trackingMiddleware(options: TrackingMiddlewareOptions = {}) {
 
           // TODO: Remove after EXPERIMENTAL_APP_SHELL AB test ends.
           if (getENV("EXPERIMENTAL_APP_SHELL")) {
-            trackExperimentViewed("client_navigation_v2")
+            trackExperimentViewed("client_navigation_v3")
```

Comment:
This method will warn and no-op if this test isn't configured in Force yet. That's in https://github.com/artsy/reaction/blob/62ccb35fcff1652e6061681c5b6ed38431982fb5/src/Artsy/Analytics/trackExperimentViewed.tsx#L6-L8